CONTRIBUTING.MD

```diff
@@ -22,9 +22,9 @@ Usage notes:
 
 ![Palette for Delta](https://github.com/Delta-Icons/android/raw/master/resources/Palette.svg) (Right-Click &rarr; Save as)
 
-![Adobe Swatch Exchange Palette](https://github.com/Delta-Icons/android/raw/master/resources/Palette.ase) (Right-Click &rarr; Save as)
+[Adobe Swatch Exchange Palette](https://github.com/Delta-Icons/android/raw/master/resources/Palette.ase) (Right-Click &rarr; Save as)
```

Comment:
GitHub isn't happy trying to render these file types, removing the `!` gets rid of those annoying broken link icons :)

![image](https://user-images.githubusercontent.com/20250090/147406968-1489f073-0c7d-4406-a9b3-7ba2f995183d.png)